coreupdate/on-premise-deployment.md

```diff
@@ -208,3 +208,36 @@ updateservicectl --server=http://localhost:8000 --user=admin --key=<previously-g
 
 You can now point your browser to `http://localhost:8000` to view the control panel.
 
+### Air-Gapped Package Management
+
+On-Premise CoreUpdate instances can be managed in a completely air-gapped environment. Internet access is not required. Below are the steps you can take to update your packages in such an environment.
+
+First you will need to decide if you want your CoreUpdate to host and serve the package files itself, or serve the files from a different fileserver.
+
+#### Option 1: Serving Package Files from CoreUpdate
+
+NOTE: this approach requires [special configuration of the CoreUpdate service](https://github.com/coreos/docs/blob/master/coreupdate/on-premise-deployment.md#package-payload-hosting).
```

Comment:
it's at [line 203](https://github.com/coreos/docs/pull/737/files#diff-6f742fe29f88016cf853319bf1e199cbR203) but it doesn't seem to be where it should. There were two "NOTE:" places.